test/integrations/database/_fixtures/Memory.fixture.ts

```diff
@@ -1,15 +1,15 @@
 
-import { SortDirections, ID } from '../../../../src/integrations/database/definitions/constants';
-import { RecordNotFound, RecordNotUpdated } from '../../../../src/integrations/database/definitions/errors';
-import { RecordData, RecordQuery, RecordSort } from '../../../../src/integrations/database/definitions/types';
-import MemoryDb from '../../../../src/integrations/database/implementations/memory/MemoryDb.js';
+import { SortDirections, ID } from '../../../../src/integrations/database/definitions/constants.js';
```

Comment:
The `.js` is not required here and is not consistent with the rest of the fixtures setup.